patches/server/0018-Multithreaded-entity-tracking.patch

```diff
@@ -280,15 +284,15 @@ index a42571cfd2c9c80df27e59db832cb64c2a64e141..e265a10781286a94d81086731ec0d333
          try {
 +            // Airplane start - multithreaded tracker
 +            if (this.trackQueue == null) this.trackQueue = new gg.airplane.structs.TrackQueue(this.world.getChunkProvider().entityTickingChunks, trackerMainQueue);
-+            if (gg.airplane.AirplaneConfig.multithreadedEntityTracker && this.world.getChunkProvider().entityTickingChunks.size() < 1000) {
++            if (gg.airplane.AirplaneConfig.multithreadedEntityTracker && (this.world.getChunkProvider().entityTickingChunks.size() * (this.world.getMinecraftServer().getPlayerCount() * 2 + 1)) > 1000) {
```

Comment:
I'm still confused why this is here. It doesn't add anything, and for a check that's supposed to return true 90% of the time anyways, it has way too much logic adding no benefit now.

Reply:
It does not return when all of players are in one place. So I made this because ticking chunks are always lower than 1000 in my case.